src/index.html

```diff
@@ -0,0 +1,28 @@
+﻿<!DOCTYPE html>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/218273877-dcf31821-2603-4c1d-93e4-89d90bcf1692.png)

- [ ] Your popup is working perfectly but as you can see the picture of the movie is going out of the screen and I cannot scroll upwards to see the full image :thinking: Kindly fix this issue :+1: because this is not according to wireframe :thinking: